data/repos.yml

```diff
@@ -222,7 +222,7 @@
 
 - repo_name: datagovuk-tech-docs
   production_url: https://guidance.data.gov.uk/
-  production_hosted_on: paas
+  production_hosted_on: gh-pages
```

Comment:
Looks good but this might not be necessary, this repo doesn't have that value set

https://github.com/alphagov/govuk-developer-docs/blob/6ae7c5cbb0ecef4fa609c7c085a1579d9af1a5fc/data/repos.yml#L552-L556